docs/arlas-ogc.md

```diff
@@ -213,3 +213,29 @@ You can query collection references metadata using the following parameters
 !!! note
    - `bbox` and `recordids` can't be used together
    - `recordids` and `q` can't be used together
+
+### Column filtering
+
+A coma-separated list of columns can be passed in request header `column-filter`. Wildcards are supported.
+
+A column filter stands for the fields that are available to a request body:
+- if a request body field doesn't belong to the column filter, a 403 is returned with the message `The field '%s' isn't available` or `The fields '%s' aren't available`;
+- only fields that belong to the column filter can be returned. 
+
+Examples of `column-filter`:
+
+- `params.city,params.country` make available `params.city` and `params.country` 
+- `params`, `params*`, `params.*`, `*params` make available `params.city`, `params.country`, `params.weight` aso.
+- `*` make all fields available
+- `*.*` make only subfields available, e.g. `params.city` and `params.country` but not `id`
+
+If no column filter, or a blank column filter is provided, then it isn't eventually used.
+
+The following WFS endpoints use this header:
+
+| Endpoint | Filtering result |
+| --------------------------------------------------- | --------------------------------------------------- |
+| /arlas/ogc/wfs/`{collection}`?request=**DescribeFeatureType** | Only fields matching this filter will be returned |
+| /arlas/ogc/wfs/`{collection}`?request=**GetFeature** | Return a 403 if one of the filter fields is not in the column filter. Otherwise only fields matching this filter will be returned |
+| /arlas/ogc/wfs/`{collection}`?request=**GetPropertyValue** | Return a 403 if property or one of the filter fields  is not in the filter. Otherwise only fields matching this filter will be returned |
+| /arlas/ogc/opensearch/`{collection} | Only fields matching this filter will be returned |
```

Comment:
Same there is two endpoints one in arlas/ogc and the other in arlas/explore :

/arlas/explore/ogc/opensearch/{collection}
/arlas/ogc/csw/opensearch